scripts/basic-workflow-tests.sh

```diff
@@ -163,6 +163,14 @@ test_convert() {
     ilab model convert
 }
 
+test_evaluate() {
+    task Evaluating a model
+    INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true 
+    HF_DATASETS_TRUST_REMOTE_CODE=true 
```

Comment:
Per the Shellchecker
```suggestion
    export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true 
    export HF_DATASETS_TRUST_REMOTE_CODE=true 
```